libraries/ui/src/constants.ts

```diff
@@ -11,6 +11,15 @@ export type Course = {
 export type CourseType = 'Crash course' | 'Self-paced' | 'In-depth course';
 
 export const COURSES: Course[] = [
+  {
+    title: 'The AI Impact',
+    description: 'A self-paced, 2-hour course designed for people with no technical background to learn how AI will reshape our world.',
+    courseType: 'Self-paced',
+    courseLength: '',
```

Comment:
```suggestion
    courseLength: '2 hours',
```

this value is required by CourseCard, where COURSES is used.

how i got there:
1. searched for "[courseLength](https://github.com/search?q=repo%3Abluedotimpact%2Fbluedot%20courseLength&type=code)" to see where else this is used
2. [libraries/ui/src/CourseCard.tsx](https://github.com/bluedotimpact/bluedot/blob/979d90e015b244bd1b2be2091475846b131dd493/libraries/ui/src/CourseCard.tsx#L19) -> `courseLength: string,` optional variables will be marked with a `?`, so that's how we can tell that this is a required variable.